core/src/main/java/org/polypheny/db/schema/Namespace.java

```diff
@@ -46,25 +46,25 @@
 
 /**
  * A namespace for tables and functions.
- *
- * A schema can also contain sub-schemas, to any level of nesting. Most providers have a limited number of levels; for
- * example, most JDBC databases have either one level ("schemas") or two levels ("database" and "catalog").
- *
+ * <p>
+ * A namespace can also contain sub-namespaces, to any level of nesting. Most providers have a limited number of levels; for
+ * example, most JDBC databases have either one level ("namespaces") or two levels ("database" and "catalog").
+ * <p>
```

Comment:
We do not (yet) support any nesting of namespaces

Reply:
Removed deprecated javadoc as we do not longer use it in this way.